submissions/slack-status/src/background.js

```diff
@@ -0,0 +1,68 @@
+async function getSlackConfig() {
+    const xoxd = (await chrome.cookies.get({
+        name: "d",
+        url: "https://app.slack.com"
+    }))?.value;
+    if (!xoxd) throw new Error("Failed to get xoxd");
```

Comment:
I don't think you should be grabbing the user's slack token from cookies without their explicit permission.

Reply:
I mean they do install a Slack status extension, and the only place I send the Slack token to is the Slack API. I'll add an authorisation modal soon though.

Reply:
Update: authorisation page has been added 
![image](https://github.com/user-attachments/assets/9fcb9999-faed-4e6c-9ab2-4d8fe2acdce8)